src/components/TabMenu/Item/Item.tsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import classNames from 'classnames';
+
+import styles from './Item.module.css';
+
+export interface ItemProps extends React.HTMLAttributes<HTMLLIElement> {
+  active?: boolean;
+}
+
+const Item: React.FC<ItemProps> = ({
+  className: customClassName,
+  active = false,
+  children,
+  onClick = () => {},
+  ...props
+}) => {
+  const className = classNames(styles.Item, customClassName, {
+    [styles.active]: active || children,
```

Comment:
this will now set the active class if "children" is truthy? will that not give false positives for inactive tabs? (where "children" would contain the node with the name of the inactive tab?)

Reply:
maybe a good first step would be to keep the functionality as is for the TS conversion, while adding some `?.` to that section so it no longer errors when there is no child node with a className prop? 
```javascript
    if (React.isValidElement(child)) {
      return child.props?.className?.indexOf('active') >= 0;
    }
```